test/standalone/Soil/climate_drivers.jl

```diff
@@ -115,6 +115,7 @@ for FT in (Float32, Float64)
             )
 
             Y, p, coords = initialize(model)
+            Δz_top, _ = ClimaLand.Domains.get_Δz(coords.subsurface.z)
```

Comment:
should we instead get `Δz_top` from the domain here to test the value we're storing, since that's how we get `Δz_top` in the code now?